src/translation/index.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import i18next from 'i18next';
+import { useTranslation, initReactI18next } from 'react-i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import cn from './cn/page.json';
+import en from './en/page.json';
+
+i18next
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    interpolation: { escapeValue: false },
+    lng: 'en',
+    fallbackLng: 'en',
+    debug: true,
+    resources: {
+      en: {
+        common: en,
```

Comment:
Why have you chosen the `common` key here?
We can optimise this. We could import separate resources for the `app` content and `help` content. We could also categorise further - please look into what would work best for us.

Reply:
The key name is from the i18n tutorial and can be changed and added more.  Can try to add the content by moving the C_HELP_CONFIG there when i figure out the better way to implement the i18next for that type of content

Reply:
From what we've discussed, i think the following taxonomy works:
- `pages` key for `config/pages.ts` translations
- `help` key for `config/help.ts` translations
- `app` for the app hierarchy. 

Regarding app - please source other projects / stack overflow to see if there are any accepted standards here.